### DISCUSSION TITLE
Breaking API changes

### DISCUSSION THREAD

**Discussion by mengstr** [pdfcpu/pdfcpu#576] [General]
Previously I had this in my code, but after an update it didn't compile until I changed `pdfcpu` to `model`. Did I miss the announcement that the public API was changed?

```
conf := pdfcpu.NewAESConfiguration("<redacted>", pw, 256)
err := api.EncryptFile(filename, filenameProtected, conf)

```

**Comment by hhrutter** [comment]
It is well known that pdfcpu is still considered *Alpha*.
I cannot change the major version as long as we are on v0.x.x
If you are using a v0.x.x version in production you are in for suprises.

**Reply by mengstr** [reply]
But but but....   It works too well to be an alpha ;)
